lms/envs/aws.py

```diff
@@ -274,6 +274,16 @@
     COMPREHENSIVE_THEME_DIR = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR')
 
 COMPREHENSIVE_THEME_DIRS = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIRS', COMPREHENSIVE_THEME_DIRS) or []
+
+# THEME_LOCALE_PATHS contain the paths to themes locale directories e.g.
+# "THEME_LOCALE_PATHS" : [
+#        "/edx/src/edx-themes/conf/locale"
+#    ],
+THEME_LOCALE_PATHS = ENV_TOKENS.get('THEME_LOCALE_PATHS', [])
+
+for theme_locale_path in THEME_LOCALE_PATHS:
+    LOCALE_PATHS += (path(theme_locale_path), )
```

Comment:
Isn't [this](https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L927) a tuple of strings?